src/grid/cubic.py

```diff
@@ -726,8 +727,19 @@ def read_grid_line(line):
             shape0, axis0 = read_grid_line(f.readline())
             shape1, axis1 = read_grid_line(f.readline())
             shape2, axis2 = read_grid_line(f.readline())
-            shape = np.array([shape0, shape1, shape2], int)
-            axes = np.array([axis0, axis1, axis2])
+            # shape = np.array([shape0, shape1, shape2], int)
+            # axes = np.array([axis0, axis1, axis2])
+            # Detect Bohr vs. Angstrom units
+            BOHR_TO_ANGSTROM = 0.529177
```

Comment:
I recommend extending the precision to 16 digits, since our calculations often rely on 64-bit floating point representations. 
Using `scipy` and `mpmath`,  the value for converting 1 Bohr to Angstrom is `0.529177210902999962`.

But since we have SciPy as a prerequisite, i think it's better to do 
```
import scipy.constants as spc
BOHR_TO_ANGSTROM = spc.value('atomic unit of length') / spc.angstrom
```